### PR TITLE
Added babel support

### PR DESCRIPTION
**Pull request by DanielMSchmidt**
Hi there,

I just gave the problem #14 and #19 a try. Somehow the previously merged commits are shown in the diff too, although they should be in master here already, I don't know why. Anyways: 

What I did:
- added build script using babel and used it as prepublish script
- checked in the `dist/` folder with the compiled files
- removed section from readme
- added myself as contributor =)

Pros:
- the user doesn't need to enable babel options in his project
- only the compiled source gets shipped
- the build command gets executed before publishing

Cons:
- I had to use Babel 5 due to an issue with decorators
  (http://phabricator.babeljs.io/T2645). IMHO moving to 6 later on should be no problem at all
- File referencing does not work, but it shouldn't be necessary in the
  first place for this library

### CODE REVIEW

File: DrawerLayout.ios.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import React from 'react-native';
-import { Animated, PanResponder, PropTypes, StyleSheet, StatusBarIOS, View, Dimensions, TouchableWithoutFeedback } from 'react-native';
```
Comment (dbanck) [inline DrawerLayout.ios.js]: This was already removed in #16, wasn't it?

File: .editorconfig [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# EditorConfig is awesome: http://EditorConfig.org
```
Comment (dbanck) [inline .editorconfig:1]: This file was already added in #18 ;)

Comment (DanielMSchmidt) [inline .editorconfig:1]: I know and it is already in Master, so I don't know why it happens to bebin the PR

**Comment by mfmendiola** [comment]
This looks great @DanielMSchmidt, thanks!  Re: the files that have already been merged to master, my only guess is that this branch is tracking an older version of master.  Rebasing should do the trick.

**Comment by DanielMSchmidt** [comment]
Hi, would you still like to have this merged?

**Comment by mfmendiola** [comment]
yep :smile:

**Comment by DanielMSchmidt** [comment]
Somehow a merge undid the changes in #16, which is the unnecessary change in this PR. But as it got merged I don't see a problem in merging this one. Is this ok for you now or would you like to have any other change?

**Comment by mfmendiola** [comment]
lgtm!

**Pull request merged**